slices/libzstd1.yaml

```diff
@@ -0,0 +1,13 @@
+# Fast lossless compression algorithm
+
```

Comment:
nit: let's remove this blank link to have the comment on top of the `package` field.